<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
        "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
    <title>KeyError: 'sum' // Werkzeug Debugger</title>
    <link rel="stylesheet" href="?__debugger__=yes&amp;cmd=resource&amp;f=style.css"
          type="text/css">
    <!-- We need to make sure this has a favicon so that the debugger does
         not by accident trigger a request to /favicon.ico which might
         change the application state. -->
    <link rel="shortcut icon"
          href="?__debugger__=yes&amp;cmd=resource&amp;f=console.png">
    <script src="?__debugger__=yes&amp;cmd=resource&amp;f=jquery.js"></script>
    <script src="?__debugger__=yes&amp;cmd=resource&amp;f=debugger.js"></script>
    <script type="text/javascript">
        var TRACEBACK = 2229055908048,
            CONSOLE_MODE = false,
            EVALEX = true,
            EVALEX_TRUSTED = false,
            SECRET = "oEexzYDb2WQTLtfFT0QS";
    </script>
</head>
<body style="background-color: #fff">
<div class="debugger">
    <h1>builtins.KeyError</h1>
    <div class="detail">
        <p class="errormsg">KeyError: 'sum'</p>
    </div>
    <h2 class="traceback">Traceback <em>(most recent call last)</em></h2>
    <div class="traceback">

        <ul>
            <li>
                <div class="frame" id="frame-2229054441176">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1997</em>,
                        in <code class="function">__call__</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">                </span>error = None</pre>
                        <pre class="line before"><span class="ws">            </span>ctx.auto_pop(error)</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">    </span>def __call__(self, environ, start_response):</pre>
                        <pre class="line before"><span class="ws">        </span>&quot;&quot;&quot;Shortcut for :attr:`wsgi_app`.&quot;&quot;&quot;</pre>
                        <pre class="line current"><span class="ws">        </span>return self.wsgi_app(environ, start_response)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def __repr__(self):</pre>
                        <pre class="line after"><span class="ws">        </span>return '&lt;%s %r&gt;' % (</pre>
                        <pre class="line after"><span class="ws">            </span>self.__class__.__name__,</pre>
                        <pre class="line after"><span class="ws">            </span>self.name,</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055706952">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1985</em>,
                        in <code class="function">wsgi_app</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">        </span>try:</pre>
                        <pre class="line before"><span class="ws">            </span>try:</pre>
                        <pre class="line before"><span class="ws">                </span>response = self.full_dispatch_request()</pre>
                        <pre class="line before"><span class="ws">            </span>except Exception as e:</pre>
                        <pre class="line before"><span class="ws">                </span>error = e</pre>
                        <pre class="line current"><span class="ws">                </span>response = self.handle_exception(e)</pre>
                        <pre class="line after"><span class="ws">            </span>except:</pre>
                        <pre class="line after"><span class="ws">                </span>error = sys.exc_info()[1]</pre>
                        <pre class="line after"><span class="ws">                </span>raise</pre>
                        <pre class="line after"><span class="ws">            </span>return response(environ, start_response)</pre>
                        <pre class="line after"><span class="ws">        </span>finally:</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055830840">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask_restful\__init__.py"</cite>,
                        line <em class="line">273</em>,
                        in <code class="function">error_router</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">        </span>if self._has_fr_route():</pre>
                        <pre class="line before"><span class="ws">            </span>try:</pre>
                        <pre class="line before"><span
                                class="ws">                </span>return self.handle_error(e)</pre>
                        <pre class="line before"><span class="ws">            </span>except Exception:</pre>
                        <pre class="line before"><span class="ws">                </span>pass  # Fall through to original handler</pre>
                        <pre class="line current"><span class="ws">        </span>return original_handler(e)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def handle_error(self, e):</pre>
                        <pre class="line after"><span class="ws">        </span>&quot;&quot;&quot;Error handler for the API transforms a raised exception into a Flask</pre>
                        <pre class="line after"><span class="ws">        </span>response, with the appropriate HTTP status code and body.</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055830672">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1540</em>,
                        in <code class="function">handle_exception</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">            </span># if we want to repropagate the exception, we can attempt to</pre>
                        <pre class="line before"><span class="ws">            </span># raise it with the whole traceback in case we can do that</pre>
                        <pre class="line before"><span class="ws">            </span># (the function was actually called from the except part)</pre>
                        <pre class="line before"><span class="ws">            </span># otherwise, we just raise the error again</pre>
                        <pre class="line before"><span class="ws">            </span>if exc_value is e:</pre>
                        <pre class="line current"><span class="ws">                </span>reraise(exc_type, exc_value, tb)</pre>
                        <pre class="line after"><span class="ws">            </span>else:</pre>
                        <pre class="line after"><span class="ws">                </span>raise e</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">        </span>self.log_exception((exc_type, exc_value, tb))</pre>
                        <pre class="line after"><span class="ws">        </span>if handler is None:</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055831680">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\_compat.py"</cite>,
                        line <em class="line">32</em>,
                        in <code class="function">reraise</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">    </span>from io import StringIO</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">    </span>def reraise(tp, value, tb=None):</pre>
                        <pre class="line before"><span class="ws">        </span>if value.__traceback__ is not tb:</pre>
                        <pre class="line current"><span
                                class="ws">            </span>raise value.with_traceback(tb)</pre>
                        <pre class="line after"><span class="ws">        </span>raise value</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>implements_to_string = _identity</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws"></span>else:</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055831456">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1982</em>,
                        in <code class="function">wsgi_app</code></h4>
                    <div class="source">
                        <pre class="line before"><span
                                class="ws">        </span>ctx = self.request_context(environ)</pre>
                        <pre class="line before"><span class="ws">        </span>ctx.push()</pre>
                        <pre class="line before"><span class="ws">        </span>error = None</pre>
                        <pre class="line before"><span class="ws">        </span>try:</pre>
                        <pre class="line before"><span class="ws">            </span>try:</pre>
                        <pre class="line current"><span class="ws">                </span>response = self.full_dispatch_request()</pre>
                        <pre class="line after"><span class="ws">            </span>except Exception as e:</pre>
                        <pre class="line after"><span class="ws">                </span>error = e</pre>
                        <pre class="line after"><span class="ws">                </span>response = self.handle_exception(e)</pre>
                        <pre class="line after"><span class="ws">            </span>except:</pre>
                        <pre class="line after"><span class="ws">                </span>error = sys.exc_info()[1]</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055831400">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1614</em>,
                        in <code class="function">full_dispatch_request</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">            </span>request_started.send(self)</pre>
                        <pre class="line before"><span
                                class="ws">            </span>rv = self.preprocess_request()</pre>
                        <pre class="line before"><span class="ws">            </span>if rv is None:</pre>
                        <pre class="line before"><span
                                class="ws">                </span>rv = self.dispatch_request()</pre>
                        <pre class="line before"><span class="ws">        </span>except Exception as e:</pre>
                        <pre class="line current"><span class="ws">            </span>rv = self.handle_user_exception(e)</pre>
                        <pre class="line after"><span class="ws">        </span>return self.finalize_request(rv)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def finalize_request(self, rv, from_error_handler=False):</pre>
                        <pre class="line after"><span class="ws">        </span>&quot;&quot;&quot;Given the return value from a view function this finalizes</pre>
                        <pre class="line after"><span class="ws">        </span>the request by converting it into a response and invoking the</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055830896">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask_restful\__init__.py"</cite>,
                        line <em class="line">273</em>,
                        in <code class="function">error_router</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">        </span>if self._has_fr_route():</pre>
                        <pre class="line before"><span class="ws">            </span>try:</pre>
                        <pre class="line before"><span
                                class="ws">                </span>return self.handle_error(e)</pre>
                        <pre class="line before"><span class="ws">            </span>except Exception:</pre>
                        <pre class="line before"><span class="ws">                </span>pass  # Fall through to original handler</pre>
                        <pre class="line current"><span class="ws">        </span>return original_handler(e)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def handle_error(self, e):</pre>
                        <pre class="line after"><span class="ws">        </span>&quot;&quot;&quot;Error handler for the API transforms a raised exception into a Flask</pre>
                        <pre class="line after"><span class="ws">        </span>response, with the appropriate HTTP status code and body.</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055831512">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1517</em>,
                        in <code class="function">handle_user_exception</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">            </span>return self.handle_http_exception(e)</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span
                                class="ws">        </span>handler = self._find_error_handler(e)</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">        </span>if handler is None:</pre>
                        <pre class="line current"><span
                                class="ws">            </span>reraise(exc_type, exc_value, tb)</pre>
                        <pre class="line after"><span class="ws">        </span>return handler(e)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def handle_exception(self, e):</pre>
                        <pre class="line after"><span class="ws">        </span>&quot;&quot;&quot;Default exception handling that kicks in when an exception</pre>
                        <pre class="line after"><span class="ws">        </span>occurs that is not caught.  In debug mode the exception will</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055831568">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\_compat.py"</cite>,
                        line <em class="line">32</em>,
                        in <code class="function">reraise</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">    </span>from io import StringIO</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">    </span>def reraise(tp, value, tb=None):</pre>
                        <pre class="line before"><span class="ws">        </span>if value.__traceback__ is not tb:</pre>
                        <pre class="line current"><span
                                class="ws">            </span>raise value.with_traceback(tb)</pre>
                        <pre class="line after"><span class="ws">        </span>raise value</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>implements_to_string = _identity</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws"></span>else:</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055944744">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1612</em>,
                        in <code class="function">full_dispatch_request</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">        </span>self.try_trigger_before_first_request_functions()</pre>
                        <pre class="line before"><span class="ws">        </span>try:</pre>
                        <pre class="line before"><span class="ws">            </span>request_started.send(self)</pre>
                        <pre class="line before"><span
                                class="ws">            </span>rv = self.preprocess_request()</pre>
                        <pre class="line before"><span class="ws">            </span>if rv is None:</pre>
                        <pre class="line current"><span
                                class="ws">                </span>rv = self.dispatch_request()</pre>
                        <pre class="line after"><span class="ws">        </span>except Exception as e:</pre>
                        <pre class="line after"><span
                                class="ws">            </span>rv = self.handle_user_exception(e)</pre>
                        <pre class="line after"><span class="ws">        </span>return self.finalize_request(rv)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def finalize_request(self, rv, from_error_handler=False):</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055945528">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\app.py"</cite>,
                        line <em class="line">1598</em>,
                        in <code class="function">dispatch_request</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">        </span># request came with the OPTIONS method, reply automatically</pre>
                        <pre class="line before"><span class="ws">        </span>if getattr(rule, 'provide_automatic_options', False) \</pre>
                        <pre class="line before"><span class="ws">           </span>and req.method == 'OPTIONS':</pre>
                        <pre class="line before"><span class="ws">            </span>return self.make_default_options_response()</pre>
                        <pre class="line before"><span class="ws">        </span># otherwise dispatch to the handler for that endpoint</pre>
                        <pre class="line current"><span class="ws">        </span>return self.view_functions[rule.endpoint](**req.view_args)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def full_dispatch_request(self):</pre>
                        <pre class="line after"><span class="ws">        </span>&quot;&quot;&quot;Dispatches the request and on top of that performs request</pre>
                        <pre class="line after"><span class="ws">        </span>pre and postprocessing as well as HTTP exception catching and</pre>
                        <pre class="line after"><span class="ws">        </span>error handling.</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055945416">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask_restful\__init__.py"</cite>,
                        line <em class="line">480</em>,
                        in <code class="function">wrapper</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">        </span>:param resource: The resource as a flask view function</pre>
                        <pre class="line before"><span class="ws">        </span>&quot;&quot;&quot;</pre>
                        <pre class="line before"><span class="ws">        </span>@wraps(resource)</pre>
                        <pre class="line before"><span class="ws">        </span>def wrapper(*args, **kwargs):</pre>
                        <pre class="line current"><span
                                class="ws">            </span>resp = resource(*args, **kwargs)</pre>
                        <pre class="line after"><span class="ws">            </span>if isinstance(resp, ResponseBase):  # There may be a better way to test</pre>
                        <pre class="line after"><span class="ws">                </span>return resp</pre>
                        <pre class="line after"><span
                                class="ws">            </span>data, code, headers = unpack(resp)</pre>
                        <pre class="line after"><span class="ws">            </span>return self.make_response(data, code, headers=headers)</pre>
                        <pre class="line after"><span class="ws">        </span>return wrapper</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055945584">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask_httpauth.py"</cite>,
                        line <em class="line">93</em>,
                        in <code class="function">decorated</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">                </span>if not self.authenticate(auth, password):</pre>
                        <pre class="line before"><span class="ws">                    </span># Clear TCP receive buffer of any pending data</pre>
                        <pre class="line before"><span class="ws">                    </span>request.data</pre>
                        <pre class="line before"><span class="ws">                    </span>return self.auth_error_callback()</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line current"><span class="ws">            </span>return f(*args, **kwargs)</pre>
                        <pre class="line after"><span class="ws">        </span>return decorated</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">    </span>def username(self):</pre>
                        <pre class="line after"><span class="ws">        </span>if not request.authorization:</pre>
                        <pre class="line after"><span class="ws">            </span>return &quot;&quot;</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055945752">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask\views.py"</cite>,
                        line <em class="line">84</em>,
                        in <code class="function">view</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">        </span>The arguments passed to :meth:`as_view` are forwarded to the</pre>
                        <pre class="line before"><span class="ws">        </span>constructor of the class.</pre>
                        <pre class="line before"><span class="ws">        </span>&quot;&quot;&quot;</pre>
                        <pre class="line before"><span class="ws">        </span>def view(*args, **kwargs):</pre>
                        <pre class="line before"><span class="ws">            </span>self = view.view_class(*class_args, **class_kwargs)</pre>
                        <pre class="line current"><span class="ws">            </span>return self.dispatch_request(*args, **kwargs)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">        </span>if cls.decorators:</pre>
                        <pre class="line after"><span class="ws">            </span>view.__name__ = name</pre>
                        <pre class="line after"><span
                                class="ws">            </span>view.__module__ = cls.__module__</pre>
                        <pre class="line after"><span
                                class="ws">            </span>for decorator in cls.decorators:</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055945136">
                    <h4>File <cite class="filename">"C:\Python35\lib\site-packages\flask_restful\__init__.py"</cite>,
                        line <em class="line">595</em>,
                        in <code class="function">dispatch_request</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">            </span>decorators = self.method_decorators</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line before"><span class="ws">        </span>for decorator in decorators:</pre>
                        <pre class="line before"><span class="ws">            </span>meth = decorator(meth)</pre>
                        <pre class="line before"><span class="ws"></span> </pre>
                        <pre class="line current"><span class="ws">        </span>resp = meth(*args, **kwargs)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">        </span>if isinstance(resp, ResponseBase):  # There may be a better way to test</pre>
                        <pre class="line after"><span class="ws">            </span>return resp</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">        </span>representations = self.representations or OrderedDict()</pre>
                    </div>
                </div>

            <li>
                <div class="frame" id="frame-2229055945808">
                    <h4>File <cite class="filename">"D:\_Main\Projetos\Pomodoro\api\activitiesAPI.py"</cite>,
                        line <em class="line">36</em>,
                        in <code class="function">get</code></h4>
                    <div class="source">
                        <pre class="line before"><span class="ws">                </span>for a2 in act2:</pre>
                        <pre class="line before"><span class="ws">                    </span>if a1['_id'] == a2['_id']:</pre>
                        <pre class="line before"><span class="ws">                        </span>a1['sum2'] = a2['sum']</pre>
                        <pre class="line before"><span class="ws">                    </span>else:</pre>
                        <pre class="line before"><span class="ws">                        </span>print(a2)</pre>
                        <pre class="line current"><span class="ws">                        </span>a2['sum2'] = a2.pop('sum')</pre>
                        <pre class="line after"><span class="ws">                        </span>act1.append(a2)</pre>
                        <pre class="line after"><span class="ws"></span> </pre>
                        <pre class="line after"><span class="ws">            </span>started3 = request.args.get('started3')</pre>
                        <pre class="line after"><span class="ws">            </span>started3 = started3 + &quot; 00:00:00&quot;</pre>
                        <pre class="line after"><span class="ws">            </span>ended3 = request.args.get('ended3')</pre>
                    </div>
                </div>
        </ul>
        <blockquote>KeyError: 'sum'</blockquote>
    </div>

    <div class="plain">
        <form action="/?__debugger__=yes&amp;cmd=paste" method="post">
            <p>
                <input type="hidden" name="language" value="pytb">
                This is the Copy/Paste friendly version of the traceback. <span
                    class="pastemessage">You can also paste this traceback into
      a <a href="https://gist.github.com/">gist</a>:
      <input type="submit" value="create paste"></span>
            </p>
            <textarea cols="50" rows="10" name="code" readonly>Traceback (most recent call last):
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1985, in wsgi_app
    response = self.handle_exception(e)
  File &quot;C:\Python35\lib\site-packages\flask_restful\__init__.py&quot;, line 273, in error_router
    return original_handler(e)
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File &quot;C:\Python35\lib\site-packages\flask\_compat.py&quot;, line 32, in reraise
    raise value.with_traceback(tb)
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File &quot;C:\Python35\lib\site-packages\flask_restful\__init__.py&quot;, line 273, in error_router
    return original_handler(e)
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File &quot;C:\Python35\lib\site-packages\flask\_compat.py&quot;, line 32, in reraise
    raise value.with_traceback(tb)
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File &quot;C:\Python35\lib\site-packages\flask\app.py&quot;, line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File &quot;C:\Python35\lib\site-packages\flask_restful\__init__.py&quot;, line 480, in wrapper
    resp = resource(*args, **kwargs)
  File &quot;C:\Python35\lib\site-packages\flask_httpauth.py&quot;, line 93, in decorated
    return f(*args, **kwargs)
  File &quot;C:\Python35\lib\site-packages\flask\views.py&quot;, line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File &quot;C:\Python35\lib\site-packages\flask_restful\__init__.py&quot;, line 595, in dispatch_request
    resp = meth(*args, **kwargs)
  File &quot;D:\_Main\Projetos\Pomodoro\api\activitiesAPI.py&quot;, line 36, in get
    a2['sum2'] = a2.pop('sum')
KeyError: 'sum'</textarea>
        </form>
    </div>
    <div class="explanation">
        The debugger caught an exception in your WSGI application. You can now
        look at the traceback which led to the error. <span class="nojavascript">
  If you enable JavaScript you can also use additional features such as code
  execution (if the evalex feature is enabled), automatic pasting of the
  exceptions and much more.</span>
    </div>
    <div class="footer">
        Brought to you by <strong class="arthur">DON'T PANIC</strong>, your
        friendly Werkzeug powered traceback interpreter.
    </div>
</div>

<div class="pin-prompt">
    <div class="inner">
        <h3>Console Locked</h3>
        <p>
            The console is locked and needs to be unlocked by entering the PIN.
            You can find the PIN printed out on the standard output of your
            shell that runs the server.
            <form>
        <p>PIN:
            <input type=text name=pin size=14>
            <input type=submit name=btn value="Confirm Pin">
            </form>
    </div>
</div>
</body>
</html>

<!--

Traceback (most recent call last):
  File "C:\Python35\lib\site-packages\flask\app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\Python35\lib\site-packages\flask\app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Python35\lib\site-packages\flask_restful\__init__.py", line 273, in error_router
    return original_handler(e)
  File "C:\Python35\lib\site-packages\flask\app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python35\lib\site-packages\flask\_compat.py", line 32, in reraise
    raise value.with_traceback(tb)
  File "C:\Python35\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python35\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python35\lib\site-packages\flask_restful\__init__.py", line 273, in error_router
    return original_handler(e)
  File "C:\Python35\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python35\lib\site-packages\flask\_compat.py", line 32, in reraise
    raise value.with_traceback(tb)
  File "C:\Python35\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python35\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Python35\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper
    resp = resource(*args, **kwargs)
  File "C:\Python35\lib\site-packages\flask_httpauth.py", line 93, in decorated
    return f(*args, **kwargs)
  File "C:\Python35\lib\site-packages\flask\views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "C:\Python35\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request
    resp = meth(*args, **kwargs)
  File "D:\_Main\Projetos\Pomodoro\api\activitiesAPI.py", line 36, in get
    a2['sum2'] = a2.pop('sum')
KeyError: 'sum'

-->
